RecoPixelVertexing/PixelTrackFitting/python/PixelTracks_cff.py

```diff
@@ -76,4 +77,26 @@
 _pixelTracksTask_lowPU.replace(pixelTracksHitQuadruplets, pixelTracksHitTriplets)
 trackingLowPU.toReplaceWith(pixelTracksTask, _pixelTracksTask_lowPU)
 
+# Use ntuple fit and substitute previous Fitter producer with the ntuple one
+from Configuration.ProcessModifiers.pixelNtupleFit_cff import pixelNtupleFit as ntupleFit
+ntupleFit.toModify(pixelTracks, Fitter = "pixelNtupletsFitter")
+_pixelTracksTask_ntupleFit = pixelTracksTask.copy()
+_pixelTracksTask_ntupleFit.replace(pixelFitterByHelixProjections, pixelNtupletsFitter)
+ntupleFit.toReplaceWith(pixelTracksTask, _pixelTracksTask_ntupleFit)
```

Comment:
I'd rather not make the python configuration even less readable that it already is.

Reply:
ugh, if by unreadable you mean being scared of seeing a `func = lambda`, perhaps so.

The benefit is the result is shorter and in fact faster to execute.

Reply:
by "unreadable" I mean that the reasonable way to do this would be something like
```python
if ntupleFit:
    pixelTracks.Fitter = "pixelNtupletsFitter"
    pixelTracksTask.replace(pixelFitterByHelixProjections, pixelNtupletsFitter)
```

however, Core Software and/or Reco don't like this approach (though I don't think it was ever explained *why*), so we are stuck with
```python
ntupleFit.toModify(pixelTracks, Fitter = "pixelNtupletsFitter")
_pixelTracksTask_ntupleFit = pixelTracksTask.copy()
_pixelTracksTask_ntupleFit.replace(pixelFitterByHelixProjections, pixelNtupletsFitter)
ntupleFit.toReplaceWith(pixelTracksTask, _pixelTracksTask_ntupleFit)
```
or, as you suggest
```
ntupleFit.toModify(pixelTracks, Fitter = "pixelNtupletsFitter")
ntupleFit.toModify(pixelTracksTask, func = lambda t: t.replace(pixelFitterByHelixProjections, pixelNtupletsFitter))
```

I do appreciate that the latter is shorter, but I also appreciate consistency: the former approach is used over 600 times in the CMSSW configuration, the latter around 60.